tldextract/tldextract.py

```diff
@@ -366,8 +369,13 @@ def fetch_file(urls):
 
     for url in urls:
         try:
-            conn = urlopen(url)
-            text = conn.read()
+            response = urlopen(url)
+            if response.info().get('Content-Encoding') == 'gzip':
```

Comment:
`urllib` makes everything so piecemeal... One day, I'm gonna break down and have this library depend on `python-requests`. ;)